docs/prod-host-setup.md

```diff
@@ -82,10 +82,10 @@ for Firecracker processes that are unresponsive, and kills them, by SIGKILL.
 
 ## Jailer Configuration
 
-For assuring secure isolation in production deployments, Firecracker should
-must be started using the `jailer` binary that's part of each Firecracker
-release, or executed under process constraints equal or more restrictive than
-those in the jailer. For more about Firecracker sandboxing please see
+For assuring secure isolation in production deployments, Firecracker should be
+started using the `jailer` binary that's part of each Firecracker release, or 
+executed under process constraints equal or more restrictive than those in the jailer.
```

Comment:
This would make more sense if we refer to the `Threat Containment` section which defines the threat model. Using the `Jailer` is an integral part o that. Users do have some leeway, but this documentation should place such situations in the "I know what I am doing" basket.

Reply:
The threat containment section is found in the design doc which we are referring to below. Maybe we can focus the attention of readers by changing the line  to
```
For more info about Firecracker sandboxing and threat containment please see [Firecracker design](design.md)
```
WDYT @sandreim?